modules/acm/policy_bundles.tf

```diff
@@ -18,12 +18,13 @@ module "policy_bundles" {
   source  = "terraform-google-modules/gcloud/google//modules/kubectl-wrapper"
   version = "~> 3.1"
 
-  for_each                = toset(var.policy_bundles)
+  # Use index as name to avoid long url or special filesystem chars
+  for_each                = { for i, v in var.policy_bundles : i => v }
```

Comment:
Would this be okay for patch release since the module key is changing which could result in recreate?

Reply:
My initial thought is this is declarative, so while it should result in a change (delete/apply) it would result in the same end cluster state.  I also considered using a `moved` block, but even if we weren't using `idx`, I didn't see a clear way to make it work with arbitrary resources.

@bharathkkb WDYT?

Reply:
Yeah it would be tricky without stable ID. I think we can just call this out in the readme as it would re apply.

Reply:
Updated the subject for the change log to clarify.